### PR TITLE
Part of #92402; fixes staticcheck failures for wsstream

### PR DESCRIPTION
**Pull request by m-Bilal**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
PR fixes issues found by staticcheck in `vendor/k8s.io/apiserver/pkg/util/wsstream`

#### Which issue(s) this PR fixes:

Part of #92402

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by m-Bilal** [comment]
I have signed the Contributor License Agreement (CLA).

File: staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go [hunk 1]
```diff
@@ -59,7 +59,8 @@ func TestRawConn(t *testing.T) {
 		defer wg.Done()
 		data, err := ioutil.ReadAll(conn.channels[0])
 		if err != nil {
-			t.Fatal(err)
+			t.Errorf("%v", err)
```
Comment (tiloso) [inline staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go:62]: `t.Error(err)` would be sufficient here and in L146. Everything else looks good to me.

**Comment by m-Bilal** [comment]
@tiloso I have updated the PR. Thanks!

File: staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go (hunk 1 above)
Comment (tiloso) [reply staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go:62]: Oh sorry, I could have been clearer: Meant that you can replace `t.Errorf("%v", err)` with `t.Error(err)`.

**Comment by m-Bilal** [comment]
@tiloso I have updated the PR. Thanks!

File: staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go [hunk 2]
```diff
@@ -59,7 +59,7 @@ func TestRawConn(t *testing.T) {
 		defer wg.Done()
 		data, err := ioutil.ReadAll(conn.channels[0])
 		if err != nil {
-			t.Fatal(err)
+			t.Error(err)
```
Comment (caesarxuchao) [inline staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go:62]: Return early here?

File: staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go [hunk 3]
```diff
@@ -141,7 +142,7 @@ func TestBase64Conn(t *testing.T) {
...
```
Comment (caesarxuchao) [inline staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go:145]: Return early here?

File: staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go (hunk 2 above)
Comment (m-Bilal) [reply staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go:62]: Oh! I think I may have misunderstood @tiloso's suggestion. @tiloso can you please confirm if there should be a `return` statement here? (In the previous comment I thought that you had asked me to remove `return` from here)

File: staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go (hunk 3 above)
Comment (m-Bilal) [reply staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go:145]: Oh! I think I may have misunderstood @tiloso's suggestion. @tiloso can you please confirm if there should be a return statement here? (In the previous comment I thought that you had asked me to remove return from here)

**Comment by m-Bilal** [comment]
@tiloso  and @caesarxuchao I have fixed the missing `return` statements and updated the PR. Please review. Thanks!

**Comment by tiloso** [comment]
Thanks a lot @m-Bilal, the PR looks good to me.

**Pull request closed without merging**